lighthouse-core/report/html/renderer/util.js

```diff
@@ -70,12 +71,42 @@ class Util {
     return output;
   }
 
+  /**
+   * @param {{score: (number|null), scoreDisplayMode: string}} audit
+   * @return {boolean}
+   */
+  static didAuditPass(audit) {
```

Comment:
maybe this shouldn't be considered if the audit "passed" (we even call out informative audits as not being able to be interpreted as pass/fail lol), but I can't think of a better word and this is the way we divide the sections in the report. Maybe add a comment that this is specifically for that distinction?

Reply:
yeah sure sg, I went back and forth on name, basically "shouldHideAudit" but not always

Reply:
actually, I could flip and do "shouldConsiderFailing" or something?